components/post_view/post_list.jsx

```diff
@@ -228,12 +270,20 @@ export default class PostList extends React.PureComponent {
     }
 
     loadMorePosts = async () => {
-        const oldestPostId = this.getOldestVisiblePostId();
+        const oldestPostId = PostList.getOldestVisiblePostId(this.state.postListIds);
 
         if (!oldestPostId) {
             // loadMorePosts shouldn't be called if we don't already have posts
             return;
         }
+        if (this.state.postListIds.length - 1 !== this.props.postListIds.length) {
+            const increasedPostListIds = this.props.postListIds.slice(0, this.state.postListIds.length + 30);
```

Comment:
Could this be done as part of `increasePostVisibility` action like in the mobile app? That would allow it to be tested as well.

Reply:
> increasePostVisibility 
In general is brittle because we just increment it with 30 when it is not the case most times. like system message which will be grouped etc